### PR TITLE
resource/aws_gamelift_game_session_queue: add notification_target argument 

### PR DESCRIPTION
**Pull request by Hinidu**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

```
$ make testacc TESTS=TestAccGameLiftGameSessionQueue PKG=gamelift

...
```

### CODE REVIEW

File: internal/service/gamelift/game_session_queue.go [hunk 1]
```diff
@@ -36,6 +36,10 @@ func ResourceGameSessionQueue() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.StringLenBetween(1, 128),
 			},
+			"notification_target": {
+				Type:     schema.TypeString,
+				Optional: true,
```
Comment (DrFaust92) [inline internal/service/gamelift/game_session_queue.go:41]: lets add arn validation as docs state that its `An SNS topic ARN that is set up to receive game session placement notifications.`

**Review by DrFaust92 — APPROVED** [review]

File: internal/service/gamelift/game_session_queue.go (hunk 1 above)
Comment (Hinidu) [reply internal/service/gamelift/game_session_queue.go:41]: I didn't find any helpers for ARN validation but found some examples of validating them using regexes. Is it what you want?

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

```console
% make testacc TESTARGS='-run=TestAccGameLiftGameSessionQueue_' PKG=gamelift ACCTEST_PARALLELISM=2
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/gamelift/... -v -count 1 -parallel 2  -run=TestAccGameLiftGameSessionQueue_ -timeout 180m
=== RUN   TestAccGameLiftGameSessionQueue_basic
--- PASS: TestAccGameLiftGameSessionQueue_basic (40.57s)
=== RUN   TestAccGameLiftGameSessionQueue_tags
--- PASS: TestAccGameLiftGameSessionQueue_tags (57.93s)
=== RUN   TestAccGameLiftGameSessionQueue_disappears
--- PASS: TestAccGameLiftGameSessionQueue_disappears (15.17s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/gamelift	124.565s
```

**Comment by ewbankkit** [comment]
@Hinidu Thanks for the contribution :tada: :clap:.

**Pull request merged**